### ISSUE TITLE
Update MirFox Firefox plugin for Firefox Quantum

### ISSUE THREAD

**Issue by tweimer** [miranda-ng/miranda-ng#1136]
The [mirfox plugin for firefox](https://addons.mozilla.org/en-US/firefox/addon/mirfox/) must be updated to support Firefox Quantum.

**Comment by tweimer** [comment]
So what does this mean for the plugin? Do we have to deprecate it?

**Comment by dartraiden** [comment]
Plugin should work in Firefox <57.  Not all users want update browser. Some users still on Firefox 52 ESR and stay on this version because it's last Firefox with Windows XP support.

**Comment by tweimer** [comment]
Any news on that? I cannot find the addon on the Firefox addon store, and without it, that plugin can not work. Shall we deprecate it?

**Comment by dartraiden** [comment]
https://github.com/JustOff/ca-archive

**Comment by tweimer** [comment]
That's an archived version of the old addon, which won't in the current Firefox?
I think at least the wiki page should be updated and the link being removed. Also there should be a warning that it isn't compatible with latest Firefox version. I fear users might be disappointed.

https://wiki.miranda-ng.org/index.php?title=Plugin:MirFox

@wsx22 Do you have plans to update that addon?

**Comment by dartraiden** [comment]
Yes, it's a addon for Firefox <57

**Comment by dartraiden** [comment]
I think we can stop supporting the MirFox plugin when Miranda drop XP support.

**Comment by tweimer** [comment]
Yes, I see that point - but there should be a note on the wiki about it.

**Comment by dartraiden** [comment]
[Done](https://wiki.miranda-ng.org/index.php?title=Plugin%3AMirFox&type=revision&diff=36230&oldid=27046)
